data/calculated_parser/functions.py

```diff
@@ -84,7 +84,7 @@ def bearing(east_vel: np.ndarray, north_vel: np.ndarray) -> np.ndarray:
                 np.abs(north_vel) < 10e-6
             )
         )
-    bearing[inds] = np.nan
+    bearing.values[inds] = np.nan
```

Comment:
I copied this bearing equation from the CSV generation code. Not sure if it's correct tho.